archive/p/python/file-io.py

```diff
@@ -0,0 +1,30 @@
+def write_file():
+
+    try:
+        out = open("file.txt", "w")
```

Comment:
Just thinking aloud here, but I think it would probably be better if we had standardized file names for this (like output.txt).

Reply:
Sounds good to me.

Reply:
Agreed. Maybe we can make a note of that in the contributing doc.